src/test/java/uk/gov/companieshouse/api/accounts/validation/AccountsValidatorTest.java

```diff
@@ -68,32 +89,50 @@
 
     @Mock
     private CompanyAccount companyAccount;
-
-    @Mock
-    private Errors errors;
     
     @Mock
     private SmallFull smallFull;
 
+    @Mock
+    private BalanceSheet currentPeriodBalanceSheet;
+
+    @Mock
+    private BalanceSheet previousPeriodBalanceSheet;
+
+    @Mock
+    private ResponseObject<CurrentPeriod> cpResponseObj;
+
+    @Mock
+    private ResponseObject<PreviousPeriod> ppResponseObj;
+
     @Mock
     private CurrentPeriodTxnClosureValidator currentPeriodTnClosureValidator;
 
     @Mock
     private PreviousPeriodTxnClosureValidator previousPeriodTnClosureValidator;
 
+    @Mock
+    private StocksTxnClosureValidator stocksTnClosureValidator;
+
     @InjectMocks
     private AccountsValidator validator;
 
     @BeforeEach
-    private void setUp() throws DataException {
+    void setUp() throws DataException {
 
         when(companyAccountService.findById(COMPANY_ACCOUNTS_ID, request)).thenReturn(companyAccountResponseObject);
         when(companyAccountResponseObject.getData()).thenReturn(companyAccount);
         when(companyAccount.getLinks()).thenReturn(createCompanyAccountLinks(true));
         
         this.validator =
```

Comment:
If we're injecting the mocks through a constructor in setup, then we do not need the InjectMocks on line 117.